app/views/view_components/preview.html.erb

```diff
@@ -1,3 +1,5 @@
+<% @render_args ||= render_args %>
```

Comment:
Why do we need this memoization? Why not reference `render_args` directly on 13?

Reply:
Hehe, I don't actually remember. I think I had a good reason but I'll try without it and see if I can remember.

Reply:
Still needs resolved ❤️

Reply:
I remember why now @joelhawksley; The `@render_args` actually isn't defined when using the `render_to_string` method in lib/view_component/system_test_helpers.rb. It seems like the method is available but no the `@` variable version. Let me know if there is a better way :)

Reply:
This is fine for now, resolving 👍🏻